ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -1016,18 +1051,8 @@ static PrjFS_Result HandleRecursivelyEnumerateDirectoryRequest(const MessageHead
     return result;
 }
 
-static PrjFS_Result HandleHydrateFileRequest(const MessageHeader* request, const char* absolutePath, const char* relativePath)
+static PrjFS_Result HydrateFile(const char* absolutePath, const char* relativePath, FsidInode fsidNode, pid_t pid, const char* procname)
```

Comment:
Nit-picking, but I just noticed the naming on the parameter here - is the "node" in `fsidNode` a typo - shouldn't that be `fsidInode`? (Or `fsidAndInode` if you want maximum accuracy) The file ID in this struct is typically referred to as an ["inode"](https://en.wikipedia.org/wiki/Inode) which is what we've called it in the rest of the kext. HFS+ actually calls it the ["CNID"](https://en.wikipedia.org/wiki/Hierarchical_File_System#Design) internally but I don't think they kept that for APFS and it also isn't called that in any of the POSIX layer APIs of macOS. [The `fsgetpath` man page](https://www.unix.com/man-page/mojave/2/fsgetpath/) also calls it an "inode".

Reply:
Nice catch, good variable names are important i'll update it!